<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialog_GRIB</class>
 <widget class="QDialog" name="QDialog_GRIB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>346</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>GRIB Request</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>331</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>320</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_req">
       <property name="text">
        <string>GRIB Request</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="spinBox_Long_Max">
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QSpinBox" name="spinBox_Long_Min">
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_minBound">
       <property name="text">
        <string>Min Bound</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QLabel" name="label_Long">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Longitude E/W</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_Max">
       <property name="text">
        <string>Max Bound</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBox_Range">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QLabel" name="label_Lat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Latitude N/S</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_Res">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>0.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.0</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_Model">
       <item>
        <property name="text">
         <string>GFS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RTOFS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Local</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ECMWG (Coming Soon)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SPIRE (Coming Soon)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QCheckBox" name="checkBox_Cloud">
       <property name="text">
        <string>Cloud Cover</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QPushButton" name="Button_Send_GRIB">
       <property name="text">
        <string>Send Request</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="data_Size_estimate">
       <property name="toolTip">
        <string>Please submit feedback</string>
       </property>
       <property name="text">
        <string>1.21 Jigawatts</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBox_Lat_Min">
       <property name="minimum">
        <number>-90</number>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Model">
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_SizeEST">
       <property name="toolTip">
        <string>Please submit feedback</string>
       </property>
       <property name="text">
        <string>Estimated GRIB Size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_Time">
       <property name="text">
        <string>Interval (hours)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="checkBox_Wave">
       <property name="text">
        <string>Waves</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="checkBox_Pressure">
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignBottom">
      <widget class="QLabel" name="label_area">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>13</height>
        </size>
       </property>
       <property name="text">
        <string>Area Selection</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Res">
       <property name="text">
        <string>Resolution (degrees)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_interval">
       <property name="text">
        <string>Time Range (Days)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QCheckBox" name="checkBox_AirT">
       <property name="text">
        <string>Air Temp</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="Button_Get_Location">
       <property name="text">
        <string>Fill Current Location</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="comboBox_Interval">
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="checkBox_CAPE">
       <property name="text">
        <string>CAPE</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="checkBox_Wind">
       <property name="text">
        <string>Wind</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QCheckBox" name="checkBox_Wing">
       <property name="text">
        <string>Wind Gust</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBox_Lat_Max">
       <property name="minimum">
        <number>-90</number>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QCheckBox" name="checkBox_Current">
       <property name="text">
        <string>Current</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="4">
      <widget class="QLineEdit" name="lineEdit_Request"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
